view/frontend/layout/checkout_index_index.xml

```diff
@@ -2,7 +2,7 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceBlock name="content">
-            <block class="Klaviyo\Reclaim\Block\Initialize" template="Klaviyo_Reclaim::analytics/initialize.phtml" />
+            <block class="Klaviyo\Reclaim\Block\Initialize" template="Klaviyo_Reclaim::analytics/initialize.phtml" name="klaviyo.checkout.initialize" />
```

Comment:
sorry for the delay on this review - can you update this to `klaviyo_reclaim.checkout.initialize` just to follow our pattern for other block names

Reply:
@cykolln , of course. Okay like that?